includes/admin/tools.php

```diff
@@ -1421,6 +1421,7 @@ function edd_tools_sysinfo_get() {
 	$return .= 'PHP Version:              ' . PHP_VERSION . "\n";
 	$return .= 'MySQL Version:            ' . $wpdb->db_version() . "\n";
 	$return .= 'Webserver Info:           ' . $_SERVER['SERVER_SOFTWARE'] . "\n";
+	$return .= 'TLS Info:                 ' . apply_filters( 'edd_get_tlsinfo', $return ) . "\n";
```

Comment:
I think this filter should perhaps be filtering a default `Unavailable` string that is transformed if it is determined TLS is available.

```php
apply_filters( 'edd_get_tlsinfo', esc_html__( 'Unavailable', 'easy-digital-downloads' ) );
```

Reply:
Gotcha, will do!